docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,214 +1,179 @@
 
-## Step 1/2. Install Teleport
+(!docs/pages/includes/tctl.mdx!)
 
-### Set up the Teleport Auth and Proxy Services
+## Step 1/4. Prepare Windows
 
-(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+In this section we'll import the Teleport certificate authority (**CA**) file
+to your Windows system, and prepare the it for passwordless access through Teleport.
 
-## Step 2/2. Run the discovery wizard
+### Import the Teleport root certificate
 
-In your web browser, access the teleport Web UI at <ScopedBlock scope={["oss", "enterprise"]}>
-`teleport.example.com`</ScopedBlock><ScopedBlock scope={["cloud"]}>
-`mytennant.teleport.sh`</ScopedBlock>. Click on your user name at the top right
-and select **Manage Access**, Select "Desktop" from the main menu, then **NEXT**:
+Use `tctl` to export the Teleport root certificate:
 
-If you already have Active Directory installed, skip to the next step. Otherwise,
-copy and paste the first command provided into a Windows PowerShell window. If
-you aren't already running AD Certificate services, copy and paste the second
-command after the first one completes and the server restarts:
+```code
+$ tctl auth export --type=windows > teleport.cer
+```
 
-![Install Active Directory](../../img/desktop-access/install-active-directory.png)
+Copy this certificate to your Windows system, if you didn't run `tctl` from there.
 
-Once the server is restarted from one or both command executions, click **NEXT**.
+### Install the Teleport service for Windows
 
-Copy and paste the provided command into a Windows PowerShell window to download
-and run the 'configure Active Directory' script:
+From the Windows system, download the [Teleport service installer](#LinkToCome) {/*Rename as needed */}
+and run it. When prompted, select the Teleport certificate file from the previous step.
+Once complete, reboot the system.
 
-![Configure Active Directory](../../img/desktop-access/configure-active-directory.png)
+## Step 2/4. Install the Teleport Desktop Service
 
-Click **NEXT**.
+On the Linux host where you will run the Desktop Service, install Teleport (if
+you plan on running the Windows Service on the same host as an existing Teleport
+service, you can skip this step):
 
-The PowerShell script will output a Teleport configuration block. Copy this
-block to a temporary location. Click **Next**.
+(!docs/pages/includes/install-linux.mdx!)
 
-On the Linux host you installed Teleport to run as the Desktop Access connector,
-edit `/etc/teleport.yaml` and paste the configuration provided by the output of
-the previous step. Review and edit the addition as necessarily for your environment:
+Enable the Desktop Service and list the Windows systems configured for
+access through Teleport in `/etc/teleport.yaml`.
 
-```yaml
+```yml
 version: v3
 teleport:
-  auth_token: 2239...c5b21
-  proxy_server: teleport.example.com:443
+  nodename: example.teleport.com
+  proxy_server: teleport-proxy.example.com:443
+windows_desktop_service:
```

Comment:
`auth_service` and `proxy_service` (and maybe `ssh_service` too, IIRC) are enabled by default, so even though we're trying to keep this config snippet small, we should probably explicitly disable them here.

Reply:
Added.